firstvoices/backend/admin/base_admin.py

```diff
@@ -154,7 +154,7 @@ def get_site_from_object(self, request):
         if not hasattr(self, "parent_site"):
             self.parent_site = None
             object_id = request.resolver_match.kwargs.get("object_id")
-            instance = self.parent_model.objects.get(id=object_id)
+            instance = self.parent_model.objects.filter(id=object_id)
             if instance and isinstance(instance, Site):
```

Comment:
Does this condition ever find a Site? I would expect it to always be a queryset. I think we need to add .first() in there somewhere

Reply:
Same timing. Haha

Reply:
Good call, thanks! I have updated this to return the first instance.